ui/utils/utils.go

```diff
@@ -177,3 +178,16 @@ func RadiusLayout(gtx layout.Context, radius int, w layout.Widget) layout.Dimens
 	call.Add(gtx.Ops)
 	return dims
 }
+
+func MarketFromAsset(asset utils.AssetType) (values.Market, error) {
```

Comment:
This name is too generic, maybe `USDMarketFromAsset?`

Reply:
I think "MarketFromAsset" good, because on the future we can use another unit like DCR-BTC...

Reply:
We should be able to rename when those other assets are added. For now, it only returns USD markets.